### PR TITLE
Add contributions List to mint modal

### PR DESCRIPTION
**Pull request by amrro**
## Linear Ticket

[PRO-492](https://linear.app/govrn/issue/PRO-492/add-a-list-of-contributions-to-the-mint-modal)

## Screencaptures
Screenshot 2022-11-14 at 3 25 11 PM

### CODE REVIEW

**Review by alexkeating — APPROVED** [review]
Looks good to me @jonathanprozzi take a look as well

**Review by jonathanprozzi — COMMENTED** [review]
This is a great addition. I have a comment on how we could potentially make it more flexible in the `text` prop.

File: apps/protocol-frontend/src/components/TextList.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import { List, ListIcon, ListItem } from '@chakra-ui/react';
+import { MdCheckCircle } from 'react-icons/all';
+import { IconType } from 'react-icons';
+
+interface TextListItemProps {
+  id: string;
+  text: string;
```
[the remark this answers is not in the document]
Comment (alexkeating) [reply apps/protocol-frontend/src/components/TextList.tsx:7]: Yea, I just ran into this with some form text, and should be easy to add

[the remark this answers is not in the document]
Comment (jonathanprozzi) [reply apps/protocol-frontend/src/components/TextList.tsx:7]: Yea exactly -- same pattern used in your recent PR

File: apps/protocol-frontend/src/components/TextList.tsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
+import { HStack, List, ListIcon, ListItem } from '@chakra-ui/react';
+import { MdCheckCircle } from 'react-icons/all';
+import { IconType } from 'react-icons';
+import React from 'react';
+
+interface TextListItemProps {
+  id: string;
+  text: string | React.ReactNode;
+  icon?: IconType;
+  iconColor?: string;
```
Comment (amrro) [inline apps/protocol-frontend/src/components/TextList.tsx:10]: @jonathanprozzi Any idea how to indicate type for color here using chakrui?

**Comment by amrro** [comment]
Great suggestion @jonathanprozzi, 
It's ready, except for small question.

**Pull request merged**